index.html

```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
+
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Stories</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">2015</a>
+          </div>
+        </div>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+      <article class="card card-3">
+        <div class="card-content"></div>
+        <img src="./images/tonic2.png" alt="Tonic" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card card-4">
+        <div class="card-content"></div>
+        <img src="./images/multipost2.png" alt="Multi-Post Stories" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Stories</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
```

Comment:
- [ ] Kindly replace `div` with `ul` tag and list the technologies with `li` tags.